spec/support/shared/shopify_request.rb

```diff
@@ -15,13 +15,17 @@
   end
 
   def mock_request(endpoint, extension)
-    file = endpoint.split('/')[0]
     url = "#{ShopifyAPI::Base.site}/#{endpoint}.#{extension}"
-    json = File.open("#{File.dirname(__FILE__)}/../data/#{file}.#{extension}")
+    json = read_file_from_endpoint(endpoint, extension)
```

Comment:
This is not really what's happening. This also means this abstractions name doesn't clue people into how to reuse this helper method.